spec/features/issues_spec.rb

```diff
@@ -49,6 +49,88 @@
             expect(page).to have_content(title)
           end
         end
+
+        describe "combine issues", js: true do
+          before do
+            # create 2 issues
+            create(:evidence)
+            create(:evidence)
+
+            visit issues_path
+          end
+
+          it "displays combine button when more than one issue selected" do
+            # no modal and no combine uissues visible
+            expect(page).to have_selector('#modal_combine_issues', visible: false)
+            expect(page).to have_selector('#combine-selected', visible: false)
+
+            # click > 1 issue checkboxes
+            page.all('input.js-multicheck').each do |checkbox|
+              checkbox.click
+            end
+
+            # then the combine button must be visible
+            expect(page).to have_selector('#combine-selected', visible: true)
+          end
+
+          it "displays combine modal when combine button clicked" do
+            # click > 1 issue checkboxes
+            page.all('input.js-multicheck').each do |checkbox|
+              checkbox.click
+            end
+
+            # click the combine button
+            find('#combine-selected').click
+
+            # then the modal should be visible
+            expect(page).to have_selector('#modal_combine_issues', visible: true)
+
+            # check that the form inside the modal has the required fields
+            expect(page).to have_selector("input[name='id']", count: 2)
+            expect(page).to have_selector("input[name='new_issue']", 1)
+            expect(page).to have_selector("input[name='sources[]']", count: 2)
+          end
+
+          it "combines issues into an existing one" do
+            # click > 1 issue checkboxes
+            page.all('input.js-multicheck').each do |checkbox|
```

Comment:
could be compressed into one line: `page.all('input.js-multicheck').each(&:click)`